docs/en/03_How_tos/redirections.md

```diff
@@ -10,13 +10,38 @@ There's a few ways developers can implement redirections on their CWP stacks:
 
 ## .htaccess
 
-Implementing in `.htaccess` is more performant than using PHP to redirect and allows more control than your standard `Director` redirection. If you are running your site with a [public/ webroot folder](https://docs.silverstripe.org/en/4/getting_started/directory_structure/), ensure that any rules are placed in `public/.htaccess`.
+Implementing in `.htaccess` is more performant than using PHP to redirect and allows more control than your standard `Director` redirection. 
 
-The below example redirects redirection.com to www.redirection.com.
+Security Headers should be placed in `public/.htaccess`
+
+More information on [HTTP Headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers) and [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy)
+
+Example
+```
+<IfModule mod_headers.c>
```

Comment:
We shouldn't include anything to do with security headers here for a couple of reasons:
a) By putting this here we're kind of making this a de-factro "official" recommendation for security headers, though other stakeholders haven't agreed that we'll do this.  So we shouldn't ;-)
b) It's in `redirections.md` :-)